pytorch_lightning/loops/utilities.py

```diff
@@ -62,6 +64,20 @@ def _extract_hiddens(training_step_output: STEP_OUTPUT, truncated_bptt_steps: in
     return hiddens
 
 
+# rank_zero_warning of type UserWarning for max_epochs if not set
+def _parse_max_epochs_and_steps(
```

Comment:
```suggestion
def _parse_loop_limits(
```
I propose this name because this function does not only take max_epochs/steps as input.